x-pack/plugins/monitoring/server/types.ts

```diff
@@ -79,10 +80,25 @@ export interface RouteDependencies {
   logger: Logger;
 }
 
+export type MonitoringRouteConfig<Params, Query, Body, Method extends RouteMethod> = {
+  // NOTE / TODO: These uppercase versions are here temporarily until all routes are converted to TS,
```

Comment:
Same here. Would be nice to have a github issue referenced so we can fill in more details there as well as hopefully not forget about it as part of the TS effort.

Reply:
Same as https://github.com/elastic/kibana/pull/130097#discussion_r850350720.